pipelines/sbom-tool-pr-build.yaml

```diff
@@ -0,0 +1,141 @@
+pr:
+  - main
+
+resources:
+    repositories:
+        - repository: 1esPipelines
+          type: git
+          name: 1ESPipelineTemplates/1ESPipelineTemplates
+          ref: refs/tags/release
+        - repository: self
+
+variables:
+  BuildConfiguration: 'Release'
+
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
+  parameters:
+    sdl:
+      sourceAnalysisPool:
+        name: sbom-windows-build-pool
+        os: windows
+    stages:
+      - stage: stage1
+        jobs:
+        - job: Job_1
+          displayName: Build (Windows)
+          pool:
+            name: sbom-windows-build-pool
+            os: windows
+          steps:
```

Comment:
It seems like lines 31-52 get repeated for each OS job. To reduce potential drift, can we put them into a separate template file and reference that template file in each OS job? A separate PR might even be able to leverage the template as part of sbom-tool-main-build.yaml

Reply:
Good idea! I've made that change, including in the main build, and kicked off a [test run of the main build](https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=29509504&view=results).